types/cavy/index.d.ts

```diff
@@ -3,25 +3,29 @@
 // Definitions by: Tyler Hoffman <https://github.com/tyler-hoffman>
 //                 Abigail McPhillips <https://github.com/AbigailMcP>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.0
+// TypeScript Version: 3.2
```

Comment:
This does mean that TypeScript 3.2 is *required* to use this package. Just want to be sure that that is your intent.

Reply:
Ahhh thanks Jesse, this is an oversight - you aren't required to use TypeScript 3.0. 

I'll revert.